### PR TITLE
runtime-rs: add missing config section for share-fs

### PR DESCRIPTION
**Pull request by Tim-0731-Hzt**
add missing config sections for share-fs

Fixes:#6020

### CODE REVIEW

**Review by studychao — APPROVED** [review]
Thanks @Tim-0731-Hzt , LGTM. Have you looked at other missing configs?

**Comment by Tim-0731-Hzt** [comment]
> Thanks @Tim-0731-Hzt , LGTM. Have you looked at other missing configs?

Yes, I'm doing this right now. If there is, I will submit another pr.

File: src/libs/kata-types/src/config/hypervisor/mod.rs [hunk 1]
```diff
@@ -767,7 +767,7 @@ pub struct SharedFsInfo {
     pub virtio_fs_extra_args: Vec<String>,
 
     /// Cache mode:
-    /// - none: Metadata, data, and pathname lookup are not cached in guest. They are always
+    /// - never: Metadata, data, and pathname lookup are not cached in guest. They are always
```
[the remark this answers is not in the document]
Comment (Tim-0731-Hzt) [reply src/libs/kata-types/src/config/hypervisor/mod.rs]: fixed

**Review by amshinde — APPROVED** [review]

File: src/libs/kata-types/src/config/hypervisor/mod.rs [hunk 2]
```diff
@@ -846,6 +850,10 @@ impl SharedFsInfo {
         if self.virtio_fs_cache.is_empty() {
             self.virtio_fs_cache = default::DEFAULT_VIRTIO_FS_CACHE_MODE.to_string();
         }
+        if self.virtio_fs_cache == *"none" {
+            warn!(sl!(),"virtio-fs cache mode `none` is deprecated since Kata Containers 2.5.0 and will be removed in the future release, please use `never` instead. For more details please refer to https://github.com/kata-containers/kata-containers/issues/4234.");
```
Comment (YushuoEdge) [inline src/libs/kata-types/src/config/hypervisor/mod.rs:854]: `,"`  -> `, "` ( add blank space )

**Review by YushuoEdge — APPROVED** [review]
LGTM to me except for a typo.

File: src/libs/kata-types/src/config/hypervisor/mod.rs [hunk 3]
```diff
@@ -779,6 +779,10 @@ pub struct SharedFsInfo {
     #[serde(default)]
     pub virtio_fs_cache_size: u32,
 
+    /// Default size of virtqueues
+    #[serde(default)]
+    pub virtio_fs_queue_size: u32,
```
Comment (bergwolf) [inline src/libs/kata-types/src/config/hypervisor/mod.rs:784]: This is unused?

Comment (Tim-0731-Hzt) [reply src/libs/kata-types/src/config/hypervisor/mod.rs:784]: This is not used for Dragonball currently, but this file may be used for other hypervisors.

File: src/libs/kata-types/src/config/hypervisor/mod.rs (hunk 2 above)
Comment (Tim-0731-Hzt) [reply src/libs/kata-types/src/config/hypervisor/mod.rs]: fixed

**Pull request merged**